### ISSUE TITLE
Find solution for computing distance ferry trips

### ISSUE THREAD

**Issue by han16nah** [pledge4future/co2calculator#98]
Computing the distance of ferry trips between two known ports is tricky. 

Steps:
* evaluate, whether there is actually a port at the specified location/address
* get the ferry route and derive its distance (there are some APIs for that, but none of them is free)

Sometimes, a ferry route may be similar to the great circle distance (our current approach), but sometimes, it may be much higher. It is difficult to work with/to find a universally valid detour coefficient.

An option could be retrieving ferry routes with the ohsome API (see below). These routes could be compared with input start and destination and - in case of a match - used to obtain the distance.

```python
import requests
import json
URL = 'https://api.ohsome.org/v1/elements/geometry'
data = {"bboxes": "-180,-90,180,90", "time": "2022-01-01", "filter": "route=ferry and type:way"}
response = requests.post(URL, data=data)
with open('ferry_routes.json', 'w', encoding='utf-8') as f:
    json.dump(response.json(), f, indent=4)
```

This requires testing (also in terms of runtime).

**Comment by han16nah** [comment]
Check openrouteservice for this (includes ferry by default)

**Comment by han16nah** [comment]
Closing because implemented in #123 
But definitely needs thorough testing and might need to be optimized
